Sources/SmithyTestUtil/FormURLComparator.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.Data
+
+public enum FormURLComparator {
```

Comment:
No comparator was previously used or provided for FormURL.  Instead, body structures were decoded from both expected and actual request data.  This was hiding some incorrect serialization behavior.

Instead, this simple comparator for Form URL data is used to ensure that actual and expected Form URL bosies are equivalent.